src/app/designs/page.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import Card from '@/Components/Card';
+import Image from 'next/image';
+
+interface Product {
+    id: number;
+    imageUrl: string;
+    title: string;
+    price: string;
+    rating: string;
+}
+
+const products: Product[] = [
+    {
+        id: 1,
+        imageUrl: '/design1.png',
+        title: 'Spread Collar Shirt',
+        price: '48.99',
+        rating: '5.0',
+    },
+    {
+        id: 2,
+        imageUrl: '/design2.png',
+        title: 'White Solid Formal Shirt',
+        price: '39.0',
+        rating: '4.9',
+    },
+    {
+        id: 3,
+        imageUrl: '/design3.png',
+        title: 'Gray Solid Padded Jacket',
+        price: '39.0',
+        rating: '4.8',
+    },
+    {
+        id: 4,
+        imageUrl: '/design4.png',
+        title: 'Printed Loose T-shirt',
+        price: '39.99',
+        rating: '4.7',
+    },
+    {
+        id: 1,
+        imageUrl: '/design1.png',
+        title: 'Spread Collar Shirt',
+        price: '48.99',
+        rating: '5.0',
+    },
+    {
+        id: 2,
+        imageUrl: '/design2.png',
+        title: 'White Solid Formal Shirt',
+        price: '39.0',
+        rating: '4.9',
+    },
+    {
+        id: 3,
+        imageUrl: '/design3.png',
+        title: 'Gray Solid Padded Jacket',
+        price: '39.0',
+        rating: '4.8',
+    },
+    {
+        id: 4,
+        imageUrl: '/design4.png',
+        title: 'Printed Loose T-shirt',
+        price: '39.99',
+        rating: '4.7',
+    },
+];
+
+const Home = () => {
+    return (
+        <div className="py-8 px-28">
+            <h2 className="text-3xl font-roboto font-medium text-[#224F34] text-center mb-8">Our designs</h2>
+            <div className="grid grid-cols-1 sm:grid-cols-2 lg:grid-cols-4 gap-6">
```

Comment:
The amount of padding seems a little excessive compared to the original design. In the original design, there is more space near the walls and sides compared to the centres. The hope is to include extra design aspects in those parts, but for the timebeing theyre blank. It might be better if the padding is reduced to mimic the design.

Reply:
fixed the padding with 80% width instead of hardcoded padding.